text/0000-maturity-of-otel.md

```diff
@@ -0,0 +1,149 @@
+
+#### Development
+
+Not all pieces of the component are in place yet, and it might not be available for users yet. Bugs and performance issues are expected to be reported. User feedback is desired, especially regarding the user experience (configuration options, component observability, technical implementation details, ...). Configuration options might break often depending on how things evolve. The component SHOULD NOT be used in production.
+
+#### Alpha
+
+This is the default level: any components with no explicit maturity level should be assumed to be “Alpha”. The component is ready to be used for limited non-critical production workloads, and the authors of this component welcome user feedback. Bugs and performance problems are encouraged to be reported, but component owners might not work on them immediately. The configuration options might often change without backward compatibility guarantees.
+
+#### Beta
+
+Same as Alpha, but the interfaces (API, configuration, generated telemetry) are treated as stable whenever possible. While there might be breaking changes between releases, component owners should try to minimize them. A component at this stage is expected to have had exposure to non-critical production workloads already during its Alpha phase, making it suitable for broader usage.
+
+#### Stable
+
+The component is ready for general availability. Bugs and performance problems should be reported, and there's an expectation that the component owners will work on them. Breaking changes, including configuration options and the component's output, are not expected to happen without prior notice unless under special circumstances.
+
+#### Deprecated
+
+Development of this component is halted, and no further support will be provided. No new versions are planned, and the component might be removed from its included distributions. Note that new issues will likely not be worked on except for critical security issues. Components that are included in distributions are expected to exist for at least two minor releases or six months, whichever happens later. They also MUST communicate in which version they will be removed.
+
+#### Unmaintained
+
+A component identified as unmaintained does not have an active code owner. Such components may have never been assigned a code owner, or a previously active code owner has not responded to requests for feedback within 6 weeks of being contacted. Issues and pull requests for unmaintained components SHOULD be labeled as such. After 6 months of being unmaintained, these components MAY be deprecated. Unmaintained components are actively seeking contributors to become code owners.
+
+### Specification requirements
+
+The specification should declare all requirements for a specified component to be declared stable. When implementations are still missing features required by the specification, they can be at most Alpha status.
```

Comment:
Should the new features be marked as "tier-1", they have to first be "candidate tier-1", allowing implementations to catch-up. Once the tier-1 implementations are caught-up, it can be promoted to "tier-1". If an implementation is taking an unreasonably long time, GC/TC has to decide to either not promote the new feature to tier-1, or to drop the implementation from tier-1.

Reply:
@reyang, do you think further clarification is needed on this?